incuna-sass/_normalize.sass

```diff
@@ -120,14 +120,14 @@ sub
 // Embedded content
 
 // Correct overflow displayed oddly in IE 9.
-@if support-legacy-browser("ie", "9")
+.lt-ie10 &
```

Comment:
You can add conditional comments for IE9, but not IE10. So this should be fine to keep.

```html
<!--[if IE 9]>
 ...
<![endif]-->